department-of-veterans-affairs/vets-website
# Add home acreage value page

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Financial information' chapter to collect more information.

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Financial Information' chapter
4. Continue to the 'Income and assets' home acreage page page
5. Verify that the 'Income and assets' home acreage value page is the next page when 'Yes' is selected
6. Verify that the 'Income and assets' home acreage page is NOT the next page when 'No' is selected
7. Verify the layout and content on the home acreage page
8. Verify the field is not required to continue
9. Enter a valid currency value into the field and validate that an error is not displayed
10. Enter a non currency value ('$1,000.00', 'abc', '-500', '1.234',  '1.234.567') into the field and validate that an error message is displayed
11. Enter a billion dollars or larger (1000000000) and validate an error is displayed
12. Verify that the form data is saved to the redux store and submitted to the api

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70425)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Home Ownership Acreage Page | Valid | Invalid | Too large |
| ------ | ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_financial_home-ownership_acres_value](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/c43faaa8-5e25-4379-ba7a-def9c415a055)|![localhost_3001_pension_application_527EZ_financial_home-ownership_acres_value (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/5ddd5754-3308-40c5-9eb3-1ff16717f401)|![localhost_3001_pension_application_527EZ_financial_home-ownership_acres_value (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/0976fa86-37e0-4732-bd2f-a8364e52f176)|![localhost_3001_pension_application_527EZ_financial_receives-income](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/eb6604d9-806d-41b2-bf6b-ee0e6374fb51)


## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/70425)

### Quality Assurance & Testing

### Error Handling


### Authentication
